metrics/lib/common.bash

```diff
@@ -18,12 +18,17 @@ SCRIPT_PATH=$(dirname $(readlink -f $0))
 RESULT_DIR="${SCRIPT_PATH}/../results"
 LIB_DIR="${SCRIPT_PATH}/../lib"
 
+# Set variables to reasonable defaults if unset or empty
+DOCKER_EXE="${DOCKER_EXE:-docker}"
+RUNTIME="${RUNTIME:-cor}"
```

Comment:
maybe set the default to the CC3.x runtime name rather than the CC2.x runtime name.